.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+<!-- Please provide all the information below.  -->
+
+### Issue describing the changes in this PR
+
+Resolves #issue_for_this_pr
```

Comment:
Fabio made the comment that "For GH keywords, most of our templates standardize on using them with no casing variation."

So I think we're leaving this as a lowercase to stay consistent 

```suggestion
resolves #issue_for_this_pr
```

Reply:
fixed.